### PR TITLE
Sprint12 05 a46725 block device logging unused inputs

### PR DESCRIPTION
**Pull request by arangamani**
Added in the log that 'volume_size and stripe_count are not used in Rackspace' for rackspace cloud provider. It has been tested with both Rackspace server and a non-Rackspace server.

### CODE REVIEW

File: cookbooks/block_device/recipes/default.rb [hunk 1]
```diff
@@ -24,6 +24,10 @@ class Chef::Resource::BlockDevice
   not_if { node[:platform] == "redhat" }
 end
 
+log "Inputs volume_size and stripe_count are not used in Rackspace cloud provider" do
```
Comment (ekargard) [inline cookbooks/block_device/recipes/default.rb]: Do we have a support doc link that talks about this?  I would ask dean/david N.  That or briefly explanin why "Rackspace does not support volumes, only ROS, so inputs  volume_size and stripe_count are ignored."

Comment (lopaka) [inline cookbooks/block_device/recipes/default.rb]: http://support.rightscale.com/09-Clouds/Rackspace_Hosting#Unsupported_Features

**Comment by lopaka** [comment]
Looks good after adding the link.  Ready for merge.

**Comment by arangamani** [comment]
Thanks, Lopaka. Merging it now.

**Pull request merged**